Quick choice dual columns

ericrsmith35 opened the pull request:
Added an option to display the visual textboxes in a single or dual columns and cleaned up the html code

alexed1 commented on flow_screen_components/quickChoiceFSCproject/force-app/main/default/lwc/quickChoiceFSC/quickChoiceFSC.js-meta.xml:
```diff
@@ -8,21 +8,22 @@
+            <property name="value" label="Value" type="String"/>
+            <property name="choiceValues" label="Choice Values (String Collection)" type="String[]"/>
+            <property name="choiceLabels" label="Choice Labels (String Collection)" type="String[]"/>
+            <property name="choiceIcons" label="Choice Icons (String Collection)" type="String[]"/>
```
Let's prepend "Visual Mode - " in the labels of properties that are specific to your mode for clarity. I'll do it.

alexed1 commented:
I merged this but then ran into a cryptic error that wouldn't let me push the resulting code to a scratch org. So I reverted the merge, which means master does not have these changes.

Pull the latest master and confirm that the combined code pushes to a new scratch org without problem. Then create a new pull request and I'll try again. 

Let's rename your mode from "Visual" to "Card" for clarity. 

Note that I've removed the field ObjectAndFilename. It was duplicative  because we can solve the problem with separate object and filename attributes

ericrsmith35 commented:
Did you run into an issue with the managed package or with the source code?

On Sun, Jan 12, 2020 at 6:54 PM Alex Edelstein <notifications@github.com>
wrote:

> I merged this but then ran into a cryptic error that wouldn't let me push
> the resulting code to a scratch org. So I reverted the merge, which means
> master does not have these changes.
>
> Pull the latest master and confirm that the combined code pushes to a new
> scratch org without problem. Then create a new pull request and I'll try
> again.
>
> Let's rename your mode from "Visual" to "Card" for clarity.
>
> Note that I've removed the field ObjectAndFilename. It was duplicative
> because we can solve the problem with separate object and filename
> attributes
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alexed1/LightningFlowComponents/pull/247?email_source=notifications&email_token=AF4UHXY7DCIFVTQFB2VYGZ3Q5OUURA5CNFSM4KFYSWZKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIXH3EQ#issuecomment-573472146>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF4UHXZUMY25VFYUZBEX34TQ5OUURANCNFSM4KFYSWZA>
> .
>
